border0/resource_connector_test.go

```diff
@@ -0,0 +1,93 @@
+package border0_test
+
+import (
+	"testing"
+
+	border0client "github.com/borderzero/border0-go/client"
+	"github.com/borderzero/terraform-provider-border0/mocks"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+var initialConnectorConfig = `
+resource "border0_connector" "unit_test" {
+  name = "unit-test-connector-1"
+  description = "connector created from terraform unit test"
+}
```

Comment:
in the future would be nice to have a test for the ssh server feature. but not now.

Reply:
Yeah, agreed.